addons/mail/static/src/models/autocomplete_input_view/autocomplete_input_view.js

```diff
@@ -34,7 +28,7 @@ registerModel({
             this.delete();
         },
         onComponentUpdate() {
-            if (this.doFocus) {
+            if (this.doFocus && this.component) {
```

Comment:
How does it happen that the component is not set in the update hook from the same component?
Checking for `this.exists()` would make sense, but for component it's really strange.

Reply:
I'll check again. Had this crash but didn't check much further.